frontend/lib/resolution/resolution-queries.cpp

```diff
@@ -144,7 +144,12 @@ const ResolutionResultByPostorderID& resolveModule(Context* context, ID id) {
           int lastId = firstId + stmtId.numContainedChildren();
           for (int i = firstId; i <= lastId; i++) {
             ID exprId(stmtId.symbolPath(), i, 0);
-            result.byIdExpanding(exprId) = resolved.byId(exprId);
+            const ResolvedExpression* reToCopy = resolved.byIdOrNull(exprId);
+            if (reToCopy) {
+              result.byId(exprId) = *reToCopy;
+            } else {
+              result.byId(exprId);
```

Comment:
I would leave a comment here as to why you're calling `byId` but throwing away the result (in fact, I would imagine at least one of the compilers will yell at you about discarding the value here). I'm assuming you're trying to create a default RRPID here by making use of how `operator[]` behaves.

Also, you could write this as:
```suggestion
            auto& re = result.byId(exprId);
            if (auto reToCopy = resolved.byIdOrNull(exprId)) {
              re = *reToCopy;
```

Reply:
Yes, I mean to have it create a default `ResolvedExpression`. The way you suggested writing it is definitely better and I will change it to that -- I think this was the result of some experimenting I forgot to put back.